documentation/examples/prometheus-kubernetes.yml

```diff
@@ -28,11 +28,9 @@ scrape_configs:
     # insecure_skip_verify: true
   bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
 
-  kubernetes_sd_configs:
-  - api_servers:
-    - 'https://kubernetes.default.svc'
-    in_cluster: true
-    role: apiserver
+  static_configs:
+  - targets:
+    - kubernetes.default.svc
```

Comment:
I'll add a comment saying this will only work for single API server setups, for HA server setups each API server should be listed separately as each should be separate scrape target?

> In general, endpoints discovery relabeling on namespace + service name would work correctly.

What do you mean? For API servers? I don't understand that I'm afraid.